exporters/prometheus/testdata/counter_already_unit_suffix.txt

```diff
@@ -0,0 +1,10 @@
+# HELP "foo.seconds_total" a simple counter
```

Comment:
This testdata file does not seem to be used. Should we remove it?

Reply:
Thanks, removed.